crates/db_schema/src/impls/community.rs

```diff
@@ -324,22 +325,8 @@ impl Bannable for CommunityPersonBan {
 }
 
 impl CommunityFollower {
-  pub fn to_subscribed_type(follower: &Option<Self>) -> SubscribedType {
-    match follower {
-      Some(f) => {
-        if f.pending {
-          SubscribedType::Pending
-        } else {
-          SubscribedType::Subscribed
-        }
-      }
-      // If the row doesn't exist, the person isn't a follower.
-      None => SubscribedType::NotSubscribed,
-    }
-  }
-
-  pub fn select_subscribed_type() -> dsl::Nullable<community_follower::pending> {
-    community_follower::pending.nullable()
+  pub fn select_subscribed_type() -> dsl::Nullable<community_follower::state> {
```

Comment:
These functions are so strange to me. Why not just impl Crud for communityfollower, or a simple read.

Reply:
It needs to be able to convert a nonexistent follower row into `SubscribedType::NotSubscribed`. That probably doesnt work with a simple read. Anyway I didnt implement this, just made some adjustments.